orttraining/orttraining/training_api/include/module.h

```diff
@@ -101,7 +101,7 @@ struct Module {
   std::vector<OrtValue> weights_;
   std::vector<OrtValue> gradients_;
   bool accumulate_gradient_ = false;
-  const std::unordered_map<std::string, std::shared_ptr<Parameter>>& named_parameters_;
+  const std::unordered_map<std::string, std::shared_ptr<Parameter>> named_parameters_;
```

Comment:
Why is this change required?

Reply:
Initializing the optimizer using the model's parameters did not work previously, needed that change to fix it.